tests/dummy/app/controllers/query.ts

```diff
@@ -1,14 +1,20 @@
 import { action } from '@ember/object';
 import Controller from '@ember/controller';
+import {
+  query,
+  orderBy,
+  limit,
+} from 'ember-cloud-firestore-adapter/firebase/firestore';
 
-import firebase from 'firebase/compat/app';
+import type firebase from 'firebase/compat/app';
```

Comment:
No big deal here, up to you if you want to change it. But I follow Python best practice for import order, which I quote:

> Imports should be grouped in the following order:
>
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

Which if I would do would be:

```typescript
import { action } from '@ember/object';
import Controller from '@ember/controller';

import type firebase from 'firebase/compat/app';

import {
  query,
  orderBy,
  limit,
} from 'ember-cloud-firestore-adapter/firebase/firestore';
```

Reply:
Lmk if I didn't do it right, I was unaware of this import organization style but I like it

Reply:
I don't want to stress on it too much since it's an opinionated thing, but if you insist, `CollectionReference` is a third party import so it should be in the middle.

```javascript
import { action } from '@ember/object';
import Controller from '@ember/controller';

import { CollectionReference } from 'firebase/firestore';

import {
  query,
  orderBy,
  limit,
} from 'ember-cloud-firestore-adapter/firebase/firestore';
```

Reply:
Ah ok I gotcha now. And no that's totally fine--I love consistency and am very much interested in understanding the patterns you are adhering to so I can follow them